src/edit.rs

```diff
@@ -19,10 +19,11 @@
 
 #![allow(clippy::unreadable_literal)] // bitflags are easier to read without underscores!!
 
-use crate::{fatal, profile_path, utils::get_name1};
+use crate::fatal;
+use crate::profile::{Profile, ProfileFlags};
 use bitflags::bitflags;
 use clap::ArgMatches;
-use log::{debug, info, warn};
+use log::{debug, error, info, warn};
```

Comment:
`error` is no longer used.

BTW: Why does `cargo clippy -- -Dwarnings` not fail in the CI?

Reply:
Apparently it does not fail but it warns which is passed by the CI.